tests/passing/small/sqrt.bril

```diff
@@ -0,0 +1,24 @@
+# ARGS: 16
```

Comment:
nit: sqrt might be a misleading name since the simplified example isn't doing sqrt

Reply:
Went ahead and made the test more real